releng/com.zeligsoft.papyrus-cx.axcioma.rcp.product/papyrus-cx-axcioma.product

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?pde version="3.5"?>
 
-<product name="Papyrus CX for AXCIOMA" uid="com.zeligsoft.papyrus-cx.axcioma.rcp.product" id="com.zeligsoft.papyrus-cx.axcioma.rcp.product" application="org.eclipse.ui.ide.workbench" version="2.2.0" useFeatures="true" includeLaunchers="true">
+<product name="Papyrus CX for AXCIOMA" uid="com.zeligsoft.papyrus-cx.axcioma.rcp.product" id="com.zeligsoft.papyrus-cx.axcioma.rcp.product" application="org.eclipse.ui.ide.workbench" version="2.5.1" useFeatures="true" includeLaunchers="true">
```

Comment:
Is this a miss from previous updates?

Reply:
Yes, we missed that in previous updates. Furthermore, I had missed updating the version number in the "About ..." dialog. I've just fixed that for both 2.5.1 and 2.6.0.